### PR TITLE
bwip-js - Add Browser implementation of the BwipJs function

### PR DESCRIPTION
**Pull request by gvanderest**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/metafloor/bwip-js#browser-usage
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by gvanderest** [comment]
Hey @andrewbranch, just curious if the bot's supposed to be editing his comment every hour?  Was there initially a countdown or something to be updated, or a constant notification to the owner?

If not, might be a place to reduce some wasted cycles.

**Comment by andrewbranch** [comment]
@gvanderest the two bot comments on this PR come from completely different processes. The perf comment (which I built) gets posted on the completion of a build, whereas the housekeeping comments run on an hourly schedule.

File: types/bwip-js/bwip-js-tests.ts [hunk 1]
```diff
@@ -37,3 +37,16 @@ bwipjs.toBuffer({
...
     }
 });
+
+// Browser canvas implementation
+const canvas = document.createElement('canvas') as HTMLCanvasElement;
+bwipjs(canvas, {
+    bcid: 'qrcode',
+    text: 'example',
+}, (err, cvs) => {
+    if (err) {
```
Comment (sheetalkamat) [inline types/bwip-js/bwip-js-tests.ts]: Add $ExpectType here instead of console.log to verify correctness of type

Comment (sheetalkamat) [inline types/bwip-js/bwip-js-tests.ts]: Add `$ExpectType` comment to verify type instead of using console.log

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/bwip-js/bwip-js-tests.ts (hunk 1 above)
Comment (gvanderest) [reply types/bwip-js/bwip-js-tests.ts]: Should I also change the existing tests?  (I was just copying them)

**Comment by andrewbranch** [comment]
Note that the scary perf numbers are mostly just due to adding the dom lib to the program.

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**